site/phrases.ejs

```diff
@@ -0,0 +1,21 @@
+# List of Warning Phrases
```

Comment:
Not sure if this file is still needed? Are we using `site/phrases.md` instead?

Reply:
Good catch!  I should have deleted that ejs file.  I'll take care of it.  Thank you!

Reply:
phrases.ejs file deleted.